tox.ini

```diff
@@ -28,24 +30,20 @@ deps =
 commands =
     zope-testrunner --test-path=src
 
-[testenv:py34-pure]
-basepython =
-    python3.4
-setenv =
-    PURE_PYTHON = 1
-    PIP_CACHE_DIR = {envdir}/.cache
 
 [testenv:py35-pure]
 basepython =
     python3.5
 setenv =
+    {[testenv]setenv}
     PURE_PYTHON = 1
     PIP_CACHE_DIR = {envdir}/.cache
 
 [testenv:py27-pure]
 basepython =
     python3.4
 setenv =
+    {[testenv]setenv}
     PURE_PYTHON = 1
     PIP_CACHE_DIR = {envdir}/.cache
```

Comment:
Incidentally, I think we can drop PIP_CACHE_DIR overrides, now that PURE_PYTHON is checked at runtime everywhere?

... assuming it is checked at runtime everywhere; I see BTrees (and zodbpickle, and zope.container, and some Zope-2-only packages like Acquisition/Persistence) still have some kind of PURE_PYTHON checks in their setup.py.  *sigh*.